proto/grp.proto

```diff
@@ -61,6 +64,41 @@ message GetGroupResponse {
   string owned_domain = 3;
 }
 
+message GetGroupUsersRequest {
+  // Request context.
+  // This contains the group ID to get the users for.
+  context.RequestContext request_context = 1;
+
+  // Membership status filter (optional).
+  GroupMembershipStatus membership_status = 2;
```

Comment:
we are not perfect about this, but I prefer to just be really verbose and automatic here and name this as the lower case version of the proto it is as long as there are no conficts. Then when you're writing the code you know exactly what it's called / what it is.

GroupMembershipStatus group_membership_status = 2;

Reply:
Done